pkg/apis/pingcap/v1alpha1/types.go

```diff
@@ -897,10 +898,6 @@ type TiDBSpec struct {
 	// Defaults to Kubernetes default storage class.
 	// +optional
 	StorageClassName *string `json:"storageClassName,omitempty"`
-	// ReadinessProbe describes actions that probe the tidb's readiness.
-	// the default behavior is like setting type as "tcp"
-	// +optional
-	ReadinessProbe *TiDBProbe `json:"readinessProbe,omitempty"`
```

Comment:
just to confirm, won't cause a rolling upgrade after putting it into `ComponentSpec`, right?

Reply:
@mikechengwei have you tested this?